### DISCUSSION TITLE
How break out until after certain number of steps.

### DISCUSSION THREAD

**Discussion by rahul-janoria** [Gremlinq/ExRam.Gremlinq#588] [General]
Hi,

I'm looking for a way to break Until after a certain number of steps.
something like this:
g.V(fromVertexId)
. repeat(both().simplePath())
.until(and(hasId(toVertexId), loops().is(lte(maxHops))))
.path()

However, I'm not able to do it with GremLinq. can someone suggest how such queries can be achieved?

**Comment by danielcweber** [comment]
That's not supported currently.

**Reply by sebastianamaro** [reply]
Hi Daniel! Is it still not supported? Do you suggest some workaround?

**Reply by danielcweber** [reply]
Still no loops-operator I am afraid.
